action_mailer-logged_smtp_delivery.gemspec

```diff
@@ -7,10 +9,10 @@ Gem::Specification.new do |gem|
   gem.files         = Dir.glob('lib/**/*') + ['README.md']
 
   gem.name          = 'action_mailer-logged_smtp_delivery'
-  gem.version       = '2.3.0'
+  gem.version       = LoggedSMTPDelivery::VERSION
```

Comment:
```suggestion
  gem.version       = ActionMailer::LoggedSMTPDelivery::VERSION
```

Reply:
The problem here is that `ActionMailer::LoggedSMTPDelivery` is not a module but a class: `class ActionMailer::LoggedSMTPDelivery < Mail::SMTP`, so `VERSION` has to go into some other namespace.

Reply:
I guess I could move `VERSION` into its own `ActionMailerLoggedSMTPDelivery` module, do you think that’s better than `LoggedSMTPDelivery`?

Reply:
Actually that’s quite a good case for having the version explicitly in the gemspec. Two alternatives come to mind:
1. Put the `VERSION` constant into the class defined in lib/action_mailer/logged_smtp_delivery.rb
2. Reopen the class. lib/action_mailer/version.rb:
```rb
class ActionMailer::LoggedSMTPDelivery < Mail::SMTP
  VERSION = "2.4.0"
end
```

Reply:
The latter is probably best, right? If we want a workflow that triggers on a `version` file?